### ISSUE TITLE
Continous Deployment V0.1

### ISSUE THREAD

**Issue by seunlanlege** [ComposableFi/composable#78]
We'd like to have special branches that once code is merged into it, we have scripts that automatically: 
- [x] If it's a client release: 
    - [x] Create a new github tagged release for the client binary signed with gpg signatures, 
    - [x] Deploy a new docker image to `composable-labs/composable:latest`
    - [ ] distribute code/binaries to node auto-update strategy
- [x] If it's a runtime release:
    - [x] create a github tagged release for the runtime wasm, signed with gpg signatures
    - [x]  auto propose new runtime wasm on picasso/composable/dali/etc
